.gitignore

```diff
@@ -25,6 +25,7 @@ packages/amplify-dynamodb-simulator/__test__/dynamodb-data
 .vscode/*
 !.vscode/settings.json
 !.vscode/extensions.json
+.idea
```

Comment:
Do we need this ?
I guess if we support IntelliJ

Reply:
Yes. I'm using WebStorm. That's where it keeps its caches and metadata.